src/components/I18nLoader.js

```diff
@@ -36,6 +36,20 @@ export class I18nLoader extends Component {
 			`../../locale/${language}/messages.json`
 		)
 
+		if (!window.Intl) {
+			await import(
+				/* webpackMode: 'lazy', webpackChunkName: 'intl-polyfill' */
+				'intl'
+			)
+			// TODO: This is also including `kde` and I've got no idea how to get rid of it
+			await import(
```

Comment:
You're waiting for three chunks to be pulled in, in sequence. You should probably be doing an `await Promise.all(...)` with the three separate import statements to let the client load all chunks in parallel. Faster client load times. You can then grab the catalog we want from the array returned from Promise.all.

Reply:
Ah shit yeah.